### PR TITLE
added lua_gettable

### PR DESCRIPTION
**Pull request by yotam5**

### CODE REVIEW

File: crates/luajit-bindings/src/ffi.rs [hunk 1]
```diff
@@ -98,7 +98,7 @@ extern "C" {
 
     // https://www.lua.org/manual/5.1/manual.html#lua_pushlightuserdata
     pub fn lua_pushlightuserdata(L: *mut lua_State, p: *mut c_void);
-
+
```
Comment (noib3) [inline crates/luajit-bindings/src/ffi.rs:101]: trailing whitespace

File: crates/luajit-bindings/src/ffi.rs [hunk 2]
```diff
@@ -152,7 +152,10 @@ extern "C" {
     pub fn lua_typename(L: *mut lua_State, tp: c_int) -> *const c_char;
 
     // Lua auxiliary library.
-
+    
+    // https://www.lua.org/manual/5.3/manual.html#lua_gettable
```
Comment (noib3) [inline crates/luajit-bindings/src/ffi.rs:156]: link the 5.1 docs, not 5.3 (Neovim uses luajit which is based off of Lua 5.1)

File: crates/luajit-bindings/src/ffi.rs [hunk 3]
```diff
@@ -152,7 +152,10 @@ extern "C" {
...
+    pub fn lua_gettable(L: *mut lua_State, tp: c_int):
```
Comment (noib3) [inline crates/luajit-bindings/src/ffi.rs:157]: the functions are ordered alphabetically, with the functions in the auxiliary library going after the ones in the API. Please reorder `lua_gettable` accordingly.

**Comment by yotam5** [comment]
noice

**Pull request closed without merging**